benchmarks/src/main.rs

```diff
@@ -42,31 +28,33 @@ use clap::{App, Arg, ArgMatches};
 use lazy_static::lazy_static;
 
 use crate::bench::{
+    bson_decode::BsonDecodeBenchmark,
+    bson_encode::BsonEncodeBenchmark,
     find_many::FindManyBenchmark,
     find_one::FindOneBenchmark,
     insert_many::InsertManyBenchmark,
     insert_one::InsertOneBenchmark,
     json_multi_export::JsonMultiExportBenchmark,
     json_multi_import::JsonMultiImportBenchmark,
     run_command::RunCommandBenchmark,
-    MAX_ITERATIONS,
+    TARGET_ITERATION_COUNT,
 };
 
 lazy_static! {
     static ref DATA_PATH: PathBuf = Path::new(env!("CARGO_MANIFEST_DIR")).join("data");
 }
 
 fn get_nth_percentile(durations: &[Duration], n: f64) -> Duration {
-    let index = (durations.len() as f64 * n / *MAX_ITERATIONS as f64) as usize;
+    let index = (durations.len() as f64 * (n / 100.0)) as usize;
     durations[std::cmp::max(index, 1) - 1]
 }
 
 fn score_test(durations: Vec<Duration>, name: &str, task_size: f64, more_info: bool) -> f64 {
-    let median = get_nth_percentile(&durations, (*MAX_ITERATIONS / 2) as f64);
+    let median = get_nth_percentile(&durations, 50.0);
     let score = task_size / (median.as_millis() as f64 / 1000.0);
     println!("TEST: {} -- Score: {}\n", name, score);
 
-    if more_info && *MAX_ITERATIONS == 100 {
+    if more_info && *TARGET_ITERATION_COUNT == 100 {
```

Comment:
Now that `get_nth_percentile` is fixed, we can drp the TARGET_ITERATION_COUNT requirement of this condition.

Reply:
Done